package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "location-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "test": "set NODE_ENV=test && jest --coverage",
+    "start": "set NODE_ENV=development && nodemon ./src/index.ts",
+    "build": "tsc -p tsconfig.json",
+    "postbuild": "copy /y .\\src\\schema\\schema.graphql .\\build\\schema\\schema.graphql && copy /y .\\aws.env .\\build\\aws.env",
+    "publish": "7z a dist/graphql.zip ./build/* && 7z u dist/graphql.zip ./node_modules",
+    "aws-deploy": "npm run build && npm test && npm run publish && terraform plan -out=.\\iac\\deployment\\test\\terraform.tfstate && terraform apply .\\iac\\deployment\\test\\terraform.tfstate"
+  },
+  "author": "mananhabib",
+  "license": "ISC",
+  "dependencies": {
+    "@aws-lambda-powertools/logger": "^1.0.2",
+    "@googlemaps/google-maps-services-js": "^3.3.16",
+    "@middy/core": "^3.1.1",
+    "apollo-datasource": "^3.3.2",
+    "apollo-server": "^3.10.0",
+    "apollo-server-core": "^3.10.0",
+    "apollo-server-lambda": "^3.10.0",
+    "aws-lambda": "^1.0.7",
+    "dotenv": "^16.0.1",
+    "graphql": "^16.5.0",
+    "nodemon": "^2.0.19",
+    "reflect-metadata": "^0.1.13",
+    "ts-node": "^10.9.1",
+    "typedi": "^0.10.0",
+    "typescript": "^4.7.4"
```

Comment:
I can see some developer dependencies defined in this section, what was the reason you decided to add them as production dependencies?

Reply:
Yeah, you are right. That's a mistake from my side. I should have used -D or --save-dev while installing these. My bad!